README.md

```diff
@@ -126,7 +128,18 @@ Dr. OtterAI will:
 1. Review your code changes
 2. Add detailed comments
 3. Create a new PR with suggested fixes (coming soon)
+  > This will be a new PR that has the fixes
```

Comment:
🔄 The comment about the new PR could be more specific. Consider adding an example of what kind of fixes might be included.